app/src/main/java/com/boolder/boolder/view/map/MapFragment.kt

```diff
@@ -118,6 +118,7 @@ class MapFragment : Fragment(), BoolderMapListener {
                     when (newState) {
                         STATE_EXPANDED -> mapViewModel.onProblemTopoVisibilityChanged(isVisible = true)
                         STATE_HIDDEN -> {
+                            binding.mapView.onBottomSheetHidden()
```

Comment:
`onTopoUnselected()` could be called here too, as the `BoulderMap`'s listener is the `MapFragment`

Reply:
As there are many ways to dismiss the topo bottom sheet, better having all the things being performed from the bottom sheet callback